internal/config/storage.go

```diff
@@ -176,6 +184,15 @@ type S3 struct {
 	PollInterval time.Duration `json:"pollInterval,omitempty" mapstructure:"poll_interval" yaml:"poll_interval,omitempty"`
 }
 
+// AZBlob contains configuration for referencing a Azure Blob Storage
+type AZBlob struct {
+	Account      string        `json:"account,omitempty" mapstructure:"account" yaml:"account,omitempty"`
+	SharedKey    string        `json:"shared_key,omitempty" mapstructure:"shared_key" yaml:"shared_key,omitempty"`
```

Comment:
```suggestion
	SharedKey    string        `json:"-" mapstructure:"shared_key" yaml:"shared_key,omitempty"`
```

We want to ignore any secret values from marshaling